tests/v9/interactions.test-d.ts

```diff
@@ -37,6 +38,10 @@ if (interaction.type === InteractionType.MessageComponent) {
 	}
 }
 
+if (interaction.type === InteractionType.ModalSubmit) {
+	expectType<APIModalSubmission | undefined>(interaction.data);
```

Comment:
Why would it be undefined... that doesn't seem right 👀 Can you make the tests be similar to the ones for ApplicationCommand? (as data shouldn't be undefined at all... and if it is, it's a bug we'll need to fix)

Reply:
ref: https://github.com/discord/discord-api-docs/pull/4486

Reply:
The docs are not clear about whether `data` is always present on modal submit interactions (~~I made that pull request~~ mostly just waiting on a staff response on the pr)